### DISCUSSION TITLE
Save Handling on a car (for other Players)

### DISCUSSION THREAD

**Discussion by Rabtos** [YimMenu/YimMenu#2187] [Q&A] [answered]
Any way to make the handling save and apply to other players?
And is it possible to save a car with a modded handling without using the profiles in the handling editor?

**Comment by Yimura** [comment, marked as answer]
On both questions no.

Because 
1. The modifications are client side and impossible to transfer to other players. If we were able to do those mods remotely R* would have a big problem on their hands.
2. The modifications you make are temporary in memory and never get saved.

**Reply by Rabtos** [reply]
OK, thanks for the fast replay Yimura!
